linked_list/c/1.linked list/160101029_1_1.c

```diff
@@ -0,0 +1,221 @@
+		}
+
+		//if alpha is found in the middle of the list
+		else if(curr->d==alpha && curr->next!=NULL)
+		{
+			count1++;						//incrementing comparisions
+
+
+			currprev->next=curr->next;		//linking prevs node to next node by skipping current node
+
+			temp->next=head1;				//adding temp at front
+			head1=temp;						//making temp as head
+			free(curr);						//deleting current node
+		}
+
+		//if alpha is not found in the list
+		else if(curr->d!=alpha && curr->next==NULL)
+		{
+			count1++;
+			curr->next=temp;				//adding node at the end
+		}
+	}
+}
+
+
+//Transpose function
+void transpose(char alpha)
+{
+	struct node *temp=(struct node *)malloc(sizeof(struct node));		//creating a temporary node
+
+	temp->d=alpha;
+	temp->next=NULL;
+
+
+	struct node *curr=head2;				//temporary pointer which points to current node
+	struct node *currprev=NULL;				//temporary pointer which points to previous node of current node
+
+	//when list is empty
+	if(head2==NULL)
+		head2=temp;
+
+	//when first element in list is alpha
+	else if(head2->d==alpha)
+	{
+		count2++;			//increment comparisions
+		return;
+	}
+
+
+	else
+	{
+		//traversing list starting from head till alpha is found or end of list
+		while(curr->d!=alpha && curr->next!=NULL)
+		{
+			count2++;					//increment comarisions
+			currprev=curr;				//storing previous node
+			curr=curr->next;			//updating current node
+		}
+
+		//if alpha is last node of the list
+		if(curr->d==alpha && curr->next==NULL)
+		{
+			count2++;					//incrementing comparision
+
+			char c;						//temporary variable to swap data
+
+			//swapping previous node data and current node data 
+			c=currprev->d;    
+			currprev->d=curr->d;
+			curr->d=c;
+		}
+
+		//if alpha is found in the middle of the list
+		else if(curr->d==alpha && curr->next!=NULL)
+		{
+			count2++;					//incrementing comparisions
+
+			char c;
+			//swapping previous node data and current node data
+			c=currprev->d;    
+			currprev->d=curr->d;
+			curr->d=c;
+		}
+
+		//if alpha is not found in the list
+		else if(curr->d!=alpha && curr->next==NULL)
+		{
+			count2++;
+			curr->next=temp;		//adding temp node at the end
+		}
+	}
+}
+
+
+//Function to print both lists
+void printlists()
+{
+	struct node *curr=head1;		//temporary pointer which points to current node
+
+	//printing Move To Front List
+	printf("MoveToFront\t");
+	//traversing through the list and printing
+	while(curr!=NULL)
+	{
+		printf("%c ",curr->d );
+		curr=curr->next;
+	}
+
+	printf("\t%d\n", count1);		//printing no. of comparisions while creating list1
+
+	curr=head2;
+
+	//printing Transpose list
+	printf("Transpose\t");
+	//traversing through the list and printing
+	while(curr!=NULL)
+	{
+		printf("%c ",curr->d );
+		curr=curr->next;
+	}
+
+
+	printf("\t%d\n",count2);		//printing no. of comparisions while creating list2
+
+}
+
+
+
+int main()
+{
+	head1=NULL;
+	head2=NULL;
+
+	//innitializing count variables
+ 	count1=0;			//comparisions for list 1
```

Comment:
Line contains following spacing inconsistencies:
- Spaces used instead of tabs.

*Origin: SpaceConsistencyBear, Section: `all.cpp`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpdqz8lpse/linked_list/c/1.linked list/160101029_1_1.c
+++ b/tmp/tmpdqz8lpse/linked_list/c/1.linked list/160101029_1_1.c
@@ -196,7 +196,7 @@
 	head2=NULL;
 
 	//innitializing count variables
- 	count1=0;			//comparisions for list 1
+	count1=0;			//comparisions for list 1
  	count2=0;			//comparisions for list 2
 
 	char alpha;			//temporary variable to store scanned data
```